packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -116,13 +120,18 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
     } = this.props;
     const { isOpen, isVisible, hasBeenOpened, modalRectangleTop } = this.state;
 
+    const layerProps = {
+      ...DefaultLayerProps,
+      ...this.props.layerProps
+    };
+
     if (!isOpen) {
       return null;
     }
 
     const classNames = getClassNames(styles, {
       theme: theme!,
-      className,
+      className: className || layerProps!.className,
```

Comment:
Why `layerProps.className` are passed to the styles of the `Modal`. This className is actually attached to the `root` section which is 2 elements deep. I think this will create a scenario where you are affecting the styles of both the `Layer` cause you are spreading `layerProps` onto it and at the same time you apply the same classname to the `classNames.root`. Is this intentional?

Reply:
Thank you for pointing this out. Somehow missed, that root is not used at the actual root <Layer>. Reverted this change